docs/tutorials/compaction-cow-mor.md

```diff
@@ -0,0 +1,170 @@
+UPDATE db_compaction.merge_on_read_table
+SET age = age + 1
+WHERE id IN (1, 2);
+```
+
+New files can be observed in the filesystem.
+
+#### Copy On Write (COW)
+<Img src="/img/tutorial/compaction-cow-mor/update-cow-fs.png" alt="update-cow-fs"/>
+
+#### Merge On Read (MOR)
+<Img src="/img/tutorial/compaction-cow-mor/update-mor-fs.png" alt="update-mor-fs"/>
+
+For Merge-on-Read (MOR), two new files are created: a delete file and a new data file.
+
+### Snapshots
+The image below illustrates that there are now two snapshots available for each of the tables.
+
+<Img src="/img/tutorial/compaction-cow-mor/snapshots-cow-1.png" alt="snapshots-cow-1"/>
+
+<Img src="/img/tutorial/compaction-cow-mor/snapshots-mor-1.png" alt="snapshots-mor-1"/>
+
+## Compaction Run 1
+We can now perform compaction on this table, as we have two snapshots available.
+
+<Img src="/img/tutorial/compaction-cow-mor/compaction-config.png" alt="compaction-config"/>
+
+:::info More details on the configuration available in the compaction job can be found [here.](https://github.com/iomete/data-compaction-job/blob/main/README.md#additional-configurations)
+:::
+
+The system tables provide detailed insights into the actions taken during compaction.
+
+<Img src="/img/tutorial/compaction-cow-mor/run-metric.png" alt="run-metric"/>
+
+### Snapshots
+Examining the available snapshots now reveals the current state.
+
+<Img src="/img/tutorial/compaction-cow-mor/snapshots-cow-2.png" alt="snapshots-cow-2"/>
+
+<Img src="/img/tutorial/compaction-cow-mor/snapshots-mor-2.png" alt="snapshots-mor-2"/>
+
+We can see that for both the tables there are now 3 snapshots available.
+
+We began with two snapshots: the first for an insert operation and the second for an update. During the compaction process, the insert snapshot expires because we configured the system to retain only one snapshot during the expiration action.
+
+The operation to rewrite data files generates a second snapshot, while the rewrite of manifest files results in a third snapshot. Consequently, we end up with a total of three snapshots.
+
+### Datafiles
+#### Copy On Write (COW)
+There are currently four files remaining at the file system level. This situation arises from the existence of a snapshot of the update operation, which includes three of these files. The fourth file is the actual compacted file referenced by the latest snapshot. The older files will be eliminated during subsequent compaction runs as we now have newer snapshots available. This process will be illustrated in the following step.
+
+<Img src="/img/tutorial/compaction-cow-mor/compaction-run1-cow-fs.png" alt="compaction-run1-cow-fs"/>
+
+#### Merge On Read (MOR)
+In a manner akin to COW, we maintain a single file that directs to the most recent snapshot; however, the majority of the files are linked to the earliest snapshot generated by the update command.
+
+<Img src="/img/tutorial/compaction-cow-mor/compaction-run1-mor-fs.png" alt="compaction-run1-mor-fs"/>
```

Comment:
It should be possible to make picture dynamic - if background is light mode, then use picture for that, if background is dark mode then use picture for that.

I think @vugardadalov has done something like that for SSO pages - https://iomete.com/resources/user-guide/single-sign-on/okta

Reply:
Will check this